Source/WebCore/rendering/GridTrackSizingAlgorithm.cpp

```diff
@@ -1708,6 +1711,9 @@ void GridTrackSizingAlgorithm::run()
     // up to this moment (before maximization) to calculate the grid container intrinsic sizes.
     computeGridContainerIntrinsicSizes();
 
+    if (m_renderGrid->isSkippedContentRoot())
+        return;
+
```

Comment:
do we need to run step #1 and #2 for skipped content?

Reply:
It used to be needed in the past, but now that I try locally I see no problem moving the check before Step 1. Let's see what the buildbots say.